.pre-commit-config.yaml

```diff
@@ -0,0 +1,83 @@
+repos:
+
+    # Clear output from jupyter notebooks so that only the input cells are committed.
+  - repo: local
+    hooks:
+      - id: jupyter-nb-clear-output
+        name: jupyter-nb-clear-output
+        description: Clear output from Jupyter notebooks.
+        files: \.ipynb$
+        stages: [commit]
+        language: system
+        entry: jupyter nbconvert --ClearOutputPreprocessor.enabled=True --inplace
+
+    # Run unit tests, verify that they pass
+  - repo: local
+    hooks:
+      - id: pytest-check
+        name: pytest-check
+        description: Run unit tests with pytest.
+        entry: pytest --cov=./src --cov-report=html
```

Comment:
Yes! Good eye, spotting the  difference here. Since the `./src` directory contains the code that will be committed, we want to run the coverage report against that.

When the workflow in github runs, the code will be installed into site-packages first. So in that case, we want to use `--cov=<package_name>`.

Reply:
I've added a comment to the python-package-template in the this file to explain the difference.

Reply:
I've added this to the python-project-template in this PR: https://github.com/lincc-frameworks/python-project-template/pull/56